python/lsst/ts/mtmount/enums.py

```diff
@@ -213,6 +213,7 @@ class CscErrorCode(enum.IntEnum):
     CONNECTION_LOST = 2  # Lost connection to low-level controller
     TELEMETRY_CLIENT_ERROR = 3  # Telemetry client failed
     AXIS_FAULT = 4  # One or more axes faulted
+    TRACK_TARGET_TIMED_OUT = 5  # A BothAxesTrackTarget command timed out
```

Comment:
Is there a SingleAxisTrackTarget command? If yes then perhaps the name for this constant isn't very well chosen, unless both that and the BothAxesTrackTarget commands use the same time out value in which case the comment may need to be updated to reflect that.